atsign-foundation/at_app_cli
# Added support for multiple templates

**- What I did**

- Updated to pull specific dependency versions when generating templates with create command (stability across future updates)
- Added support for multiple templates via a temporarily hidden flag 
- Added back a missing error check when pub adding a package

**- How I did it**

Local changes

**- How to verify it**

It's published as a release candidate and can be activated via:
```
flutter pub global activate at_app 1.1.0-rc+1
```

Then run with:
```
at_app create <name of project>
```

Finally, run the project generated by at_app to ensure that it is generating the template correctly.

**- Description for the changelog**
Added support for multiple templates.
